docs/nodes/TRANSFORMERS/IMAGE_PROCESSING/EDGE_DETECTION/a1-[autogen]/docstring.txt

```diff
@@ -0,0 +1,22 @@
+
+    Apply image smoothing operation on the input `DataContainer` class,
```

Comment:
I think this docstring is related to to Image smoothing not edge detection

Reply:
Fixed